qiskit_experiments/measurement/discriminator/discriminator_nosklearn.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+from numpy.linalg import inv
+from math import log
+
+    
+
+class LDAnosklearn():
+    
+    def __init__(self):
+        return None
+    
+    def fit(self, X, y):
+        X0 = X[y==0]
```

Comment:
this needs comments and citation for where the formula came from

Reply:
Added citation